scripts/fast-reboot

```diff
@@ -600,12 +605,15 @@ if [[ x"${DETACH}" == x"yes" && x"${ALREADY_DETACHED}" == x"" ]]; then
    exit $?
 fi
 
+sonic_asic_type=$(sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)
+if [[ "$REBOOT_TYPE" = "express-reboot" ]] && [[ "$sonic_asic_type" != "cisco-8000" ]]; then
+    echo "eXpress Boot is not supported"
+    exit "${EXIT_FAILURE}"
+fi
 
 check_conflict_boot_in_fw_update
```

Comment:
Can you add handling for `express-reboot` in `check_conflict_boot_in_fw_update`, so that there's no unexpected behavior there?